scripts/update-apk.sh

```diff
@@ -17,9 +17,9 @@ git clone --quiet --branch=apk https://fossasia:$GITHUB_API_KEY@github.com/fossa
 cd apk
 
 if [ "$TRAVIS_BRANCH" == "$PUBLISH_BRANCH" ]; then
-	/bin/rm -f *
+	/bin/rm -f open-event-master-app-debug.apk open-event-master-app-release-unsigned.apk 
```

Comment:
Wrong app names, thers's fdroid and play store version

Reply:
here open-evet-master should be changed to open-event-dev right?

Reply:
No

Reply:
My concern is that the names you have written here are wrong

Reply:
Still wrong

Reply:
is it right now?